### PR TITLE
Ingestion: ACL 2023

### PR DESCRIPTION
**Pull request by anthology-assist**
Status:
- [x] Long
- [x] Short
- [x] Industry
- [x] SRW
- [x] Tutorials
- [x] Findings

### CODE REVIEW

File: data/xml/schema.rnc [hunk 1]
```diff
@@ -12,8 +12,11 @@ MarkupText = (text | b | i | url | fixed-case | tex-math )+
 first = element first { text }
 last = element last { text }
 affiliation = element affiliation { text }
+orcid = element orcid { text }
+google-scholar = element google-scholar{ text }
+semantic-scholar = element semantic-scholar { text }
 Variant = element variant { attribute script { xsd:string }, (first? & last) }
-Person = attribute id { xsd:NCName }?, (first? & last & Variant? & affiliation? )
+Person = attribute id { xsd:NCName }?, (first? & last & Variant? & affiliation? & orcid? & google-scholar? & semantic-scholar? )
```
Comment (mbollmann) [inline data/xml/schema.rnc:19]: Just a note on this: While we should absolutely start adding ORCIDs etc., we need to be careful to not introduce inconsistencies in our data. What if a name appears several times with the same ID but different ORCIDs?

This is ultimately a danger of attaching this information to _name instances_ on papers vs. to the _person identities_ themselves.

File: data/xml/schema.rnc [hunk 2]
```diff
@@ -12,8 +12,11 @@ MarkupText = (text | b | i | url | fixed-case | tex-math )+
 first = element first { text }
 last = element last { text }
 affiliation = element affiliation { text }
+orcid = element orcid { text }
+google-scholar = element google-scholar{ text }
+semantic-scholar = element semantic-scholar { text }
```
Comment (mbollmann) [inline data/xml/schema.rnc:17]: What's the intended content of these fields? URLs (e.g. https://www.semanticscholar.org/author/Marcel-Bollmann/34887843) or IDs (34887843)?

If URLs, type should be `xsd:anyURI`, maybe even restricted by a pattern (`xsd:anyURI {pattern="https://www\.semanticscholar\.org/author/.+"}` (untested))

**Comment by ryancotterell** [comment]
Two issues on the main page:

**Comment by mbollmann** [comment]
Build will still fail due to #2501. You can try running my [bin/find_mismatched_braces.py](https://github.com/acl-org/acl-anthology/pull/2501/files#diff-6cb069864bf90d49dcb4b46bdae035630d41b0aaa356b9bfd7e101804e9917c5) script on it. Otherwise we'll have to temporarily downgrade the error to a warning until incoming data is fixed.

File: data/xml/schema.rnc (hunk 2 above)
Comment (mjpost) [reply data/xml/schema.rnc:17]: Yeah, I'm going to wait on these. I agree they should be attached to authors, and then used for disambiguation.

**Review by mjpost — APPROVED** [review]

**Comment by mbollmann** [comment]
There is no frontmatter at all for these proceedings?

**Comment by mjpost** [comment]
@mbollmann It's not ready yet, they asked me to ingest later :(

**Comment by g-vallejo** [comment]
Hi @mjpost, at the SRW we pushed yesterday a new [yaml file](https://github.com/ACL-2023/SRW/blob/main/papers.yml) in [our repo](https://github.com/ACL-2023/SRW/tree/main) to include correct summaries, given that they are currently only one sentence long. Do you think this can be included in the next build?
Thanks in advance for your help.

**Pull request merged**